src/Content/Backend/Solution/Monaco.Template.Backend.Application.Tests/Features/Company/CreateCompanyHandlerTests.cs

```diff
@@ -1,31 +1,27 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
-using System.Threading;
-using System.Threading.Tasks;
-using FluentAssertions;
+﻿using FluentAssertions;
 using Monaco.Template.Backend.Application.Features.Company;
 using Monaco.Template.Backend.Application.Infrastructure.Context;
 using Monaco.Template.Backend.Common.Tests;
 using Monaco.Template.Backend.Common.Tests.Factories;
 using Moq;
+using System.Diagnostics.CodeAnalysis;
 using Xunit;
 
 namespace Monaco.Template.Backend.Application.Tests.Features.Company;
 
 [ExcludeFromCodeCoverage]
-[Trait("Application Commands", "Create Company")]
+[Trait("Application Commands - Company", "Create")]
 public class CreateCompanyHandlerTests
 {
 	private readonly Mock<AppDbContext> _dbContextMock = new();
-	private static readonly CreateCompany.Command _command = new(It.IsAny<string>(),	// Name
-																 It.IsAny<string>(),	// Email
-																 It.IsAny<string>(),	// WebsiteUrl
-																 It.IsAny<string>(),	// Street
-																 It.IsAny<string>(),	// City
-																 It.IsAny<string>(),	// County
-																 It.IsAny<string>(),	// PostCode
-																 It.IsAny<Guid>());		// CountryId
+	private static readonly CreateCompany.Command Command = new(It.IsAny<string>(), // Name
```

Comment:
Looks like we can remove the static modifier from these fields, I think you've already updated some test classes, but there are others that still have the static modifier.

```suggestion
        private readonly CreateCompany.Command Command = new(It.IsAny<string>(), // Name
```

Reply:
Actually, I think this was intended to be left this way, just like what you generated it in the previous PR. Because if we declare it `private static readonly`, it will not allow any other modification and we always need to generate a new copy without messing with a static/shared one.